code/app/src/androidTest/java/com/example/sigma_blue/ViewListActivityUITest.java

```diff
@@ -119,79 +122,163 @@ public class ViewListActivityUITest extends UITestTools {
-//        //onView(withId(R.id.button_edit)).perform(click());
-//        // enter item info
-//        onView(withId(R.id.text_name_disp)).perform(ViewActions.typeText("iName"));
-//        closeKeyboard();
-//        onView(withId(R.id.text_value_disp)).perform(ViewActions.typeText("100"));
-//        onView(ViewMatchers.isRoot()).perform(closeSoftKeyboard());
-//        onView(withId(R.id.text_make_disp)).perform(ViewActions.typeText("Banana"));
-//        onView(ViewMatchers.isRoot()).perform(closeSoftKeyboard());
-//        onView(withId(R.id.text_model_disp)).perform(ViewActions.typeText("name"));
-//        onView(ViewMatchers.isRoot()).perform(closeSoftKeyboard());
-//        onView(withId(R.id.text_serial_disp)).perform(ViewActions.typeText("9001"));
-//        onView(ViewMatchers.isRoot()).perform(closeSoftKeyboard());
-//        onView(withId(R.id.text_comment_disp)).perform(ViewActions.typeText("comment about thing"));
-//        onView(ViewMatchers.isRoot()).perform(closeSoftKeyboard());
-//        onView(withId(R.id.text_description_disp)).perform(ViewActions.typeText("description of thing"));
-//        onView(ViewMatchers.isRoot()).perform(closeSoftKeyboard());
-//        // back to list
-//        onView(withId(R.id.button_save)).perform(click());
-//        onView(withId(R.id.button_back)).perform(click());
-//        // check if the item is displayed properly
+
+    /**
+     * Login first for setting up the following user test
+     * the first step of the all ViewList tests
+     */
+    public void testUser_Login() {
+        UITestTools.login("Temp_User1", "password");
+    }
+
+    public void testUser_Signup() {
+        UITestTools.signup("Temp_User1", "password");
+    }
+
+    /**
+     * delete user account, function should start from ViewListActivity
+     */
+    public void User_delete() {
+        onView(withId(R.id.optionButton)).perform(click());
+        onView(withText("Delete Account")).perform(click());
+        onView(withText("CONFIRM")).perform(click());
+    }
+    /**
+     * As an owner, I want to add an item to my items, with a date of purchase or acquisition, brief
+     * description, make, model, serial number (if applicable), estimated value, and comment.
+     */
+    @Test
+    public void add_item_US_01_01_01() {
+        // get the user signup
+        testUser_Signup();
+        // get the user login
+        testUser_Login();
+        // get to edit page
+        onView(withId(R.id.addButton)).perform(click());
+        //onView(withId(R.id.button_edit)).perform(click());
+        // enter item info
+        onView(withId(R.id.text_name_disp)).perform(ViewActions.typeText("iName"));
+        onView(ViewMatchers.isRoot()).perform(closeSoftKeyboard());
+        onView(withId(R.id.text_value_disp)).perform(ViewActions.replaceText("100"));
+        onView(ViewMatchers.isRoot()).perform(closeSoftKeyboard());
+        onView(withId(R.id.text_make_disp)).perform(ViewActions.typeText("Banana"));
```

Comment:
Typing text into an edit text and then closing the keyboard gets repeated so much that it should probably be factored out into a method.